### ISSUE TITLE
[Updates] Could the update channels be made clearer?

### ISSUE THREAD

**Issue by dftf-stu** [stenzek/duckstation#2122]
**Is your feature request related to a problem? Please describe.**

I try to run the latest _DuckStation_ but find it confusing which channel I should be using: I'm currently on 0.1.4042, and the "Preview" channel advises there is no update; yet the "Latest" channel (referred to as "Development", on the GitHub site) says in-app there is an update, but according to the site, the last update on that channel was published 8 days ago.

**Describe the solution you'd like**

I'd like it to be clearer which channel is the most-frequently updated -- I'm unsure between "Preview" and "Latest" which of those is the most-regularly updated?

Other apps seem to have a clearer channel structure:
- Firefox: Stable > Beta > Nightly
- Edge: Stable > Beta > Dev
- Google Chrome: Stable > Beta > Canary
- Windows 10: Stable > Monthly Preview releases > Insider Preview channel

**Describe alternatives you've considered**

Currently, I'd have to download both .ZIP packages, and run the "duckstation-qt-x64-ReleaseLTCG.exe" file in both to check what version the window titlebar reports, and keep whichever one was the more-recent.

**Additional context**

As per screenshot: in-app, I'm told the "Latest" channel has an update (though no changelog details are offered), yet on this website, the latest package for that channel was released 8 days ago...

![DuckStation_UpdateDifferences](https://user-images.githubusercontent.com/83965343/118392661-9d620980-b632-11eb-9dc3-efe56a60bff3.png)

**Comment by foxsevent** [comment]
Latest is the newest stable build while preview is basically beta build, eventually preview builds are made into the latest build after no major issues were find.

Duckstation will always say that the build you don't have is newer so people can easily swap between them without having to delete everything.

If you want the latest updated the soonest then you want the preview build thou they may contain some bugs.

**Comment by stenzek** [comment]
Yeah. It's a little messy. It's because originally there was only one update stream, so calling it "latest" made sense. But then I added preview, but I couldn't change the latest name (because existing users wouldn't be able to update).

I guess it could be described in the readme, but I'm not sure if anyone would read it anyway...

**Comment by dftf-stu** [comment]
> Latest is the newest stable build while preview is basically beta build, eventually preview builds are made into the latest build after no major issues were found.

Thanks for the clarification!
 
> DuckStation will always say that the build you don't have is newer so people can easily swap between them without having to delete everything.

Oh, I see, so essentially it lets you switch-channel.  I was not aware of that, and hence why from the UI (in my screenshot example) I assumed that "latest" was actually offering me a newer version than the preview I was on.  Hence my extra-confusion as to why in-app "latest" seemed to be offering me something the website itself wasn't.  I think this could be made-clearer.
 
> If you want the latest updated the soonest then you want the preview build

_DuckStation_ has seemed extremely-stable so-far :)

**Comment by dftf-stu** [comment]
As one simple suggestion -- how about adding some descriptive-text after each dropdown item to make it clearer what each one does?  (This shouldn't break anything internally, as you're not changing the channel names, only the form dropdown labels.)

![DuckStation_UpdateChannelDropdownText](https://user-images.githubusercontent.com/83965343/118405078-b6d47700-b66d-11eb-9a4f-9aa504174c49.png)

**Comment by dftf-stu** [comment]
Also, maybe consider adding the version number to the "Releases" page headings?

![DuckStation_AddVersionToReleasesHeading](https://user-images.githubusercontent.com/83965343/118405143-37937300-b66e-11eb-8e05-68d2bbde9029.png)

Some other projects I follow on GitHub let you know what version you'll be downloading in one of these places:
- Next to the label icon (where it says "preview" in the screenshot, below the orange "pre-release" graphic)
- In the filename of each download (examples from other GitHub projects: "MPC-HC.1.9.11.14.x64.zip", "Winfile_v10.1.4.0.zip", "PsyDoom_0.7.4_Windows_x86_64.zip" or "crispy-doom-5.10.1-win32.zip").  For _DuckStation_, the file I would download is always called "duckstation-windows-x64-release.zip".

Having a version-number on the "Releases" page would also help make-clearer which channel offers the most-recent version
